install/v1alpha1-0.7.0/cstor-volume/volume.yaml

```diff
@@ -0,0 +1,755 @@
+---
+apiVersion: openebs.io/v1alpha1
+kind: CASTemplate
+metadata:
+  name: cstor-volume-create-default-0.7.0
+spec:
+  defaultConfig:
+  - name: VolumeControllerImage
+    #value: openebs/cstor-volume-mgmt:cstor-feature-integration-ci
```

Comment:
This could be cleaned up.

Reply:
well... we will do this when we are in master....